src/applications/appeals/10182/utils/submit.js

```diff
@@ -1,18 +1,15 @@
 import moment from 'moment';
 
-import {
-  SELECTED,
-  MAX_LENGTH,
-  SUBMITTED_DISAGREEMENTS,
-  SHOW_PART3,
-} from '../constants';
+import { MAX_LENGTH, SHOW_PART3 } from '../constants';
 import { replaceSubmittedData, fixDateFormat } from './replace';
 
+import { SELECTED } from '../../shared/constants';
+
 /**
  * @typedef FormData
  * @type {Object<Object>}
  * @property {Veteran} veteran - data from prefill & profile
- * @property {ContestableIssues} contestableIssues - issues loaded from API
+ * @property {ContestableIssues} contestedIssues - issues loaded from API
```

Comment:
Should `{ContestableIssues} contestedIssues` be renamed so that it's `{ContestedIssues} contestedIssues`

Reply:
I guess we could. In the PR I'm about to open, I created a `definitions.js` file with all of these `typedef`s inside. We can discuss renaming everything once it's all in the shared folder.

Reply:
For sure. Can also tackle it with the refactor work. Keeping these in mind for when I see it again :D